core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -189,5 +193,42 @@
     return null;
   }
 
+  public static class RpcContext {
+    String mCommand;
+    String mSrcPath;
+    String mDstPath;
+    String mUser;
+    String mIp;
+    String mSrcPathOwner;
+    String mSrcPathGroup;
+    short mSrcPathMode;
+    boolean mAllowed;
+    boolean mCommitted;
+
+    public RpcContext setCommand(String command) { mCommand = command; return this; }
+    public RpcContext setSrcPath(String srcPath) { mSrcPath = srcPath; return this; }
```

Comment:
typically Alluxio code style does not recommend combining multiple statements into one line.